### ISSUE TITLE
USER ISSUE: Game not working correctly

### ISSUE THREAD

**Issue by Egorimus** [StrangeLoopGames/EcoIssues#24611]
\
**User Id: (slg234186)**
**Version:** 0.10.0.4 beta release-557

**Steps to Reproduce:**
I start my default world with "No Collaboration", and a brand new game file, after a fresh reinstall of the game.  I'm moving around with the default keys, and pushing the default hotkeys.

**Expected behavior:**
The tutorial should register that I'm pushing default keys, the hotkeys should open menus, and I should be able to place and interact with objects.

**Actual behavior:**
I can move, but those movements do not register for the tutorial.  I cannot open my menus with hotkeys, but I can with my mouse cursor.  I can drag my campsite to my hotbar, but I get no silouette to place it and no buttons work.  I cannot interact with any plants around me.

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mods are installed.

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/13944555/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/13944556/Player-prev.log)

**Comment by D3nnis3n** [comment]
Duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/24545.

Please check that issue and provide any additional info you have. Currently a DXDiag (type DXDiag into system search, run it through and add logs) would be interesting. Also if the issue persists if you use steam branch "10.0.5_testing" (right click on Eco in Steam -> Properties -> Betas -> Select from dropdown). Is this issue the same on any server, including local server?

**Comment by Egorimus** [comment]
@D3nnis3n I've ran through 10.0.5, 10.0.4 (obviously), 10.0.0, and 9.7.13, as that was the most recent patch before Update 10 you can roll back to now.  I tried logging on to a public server with 10.0.0, and was having that same result still.  However, with 9.7.13 I *can* access all the hotkeys and place/interact with things in the environment!Okay, I've ran through 10.0.5, 10.0.4 (obviously), 10.0.0, and 9.7.13, as that was the most recent patch before Update 10 you can roll back to now.  I tried logging on to a public server with 10.0.0, and was having that same result still.  However, with 9.7.13 I *can* access all the hotkeys and place/interact with things in the environment!  The tutorial still doesn't seem to notice me doing anything, such as moving, seeing the meteor, or placing my campsite.  I've included the dxdiag here, though I'm assuming you're the person who responded both here and on Steam, so I am fine with tackling this through whichever platform you find easiest/most convenient to use!
[DxDiag.txt](https://github.com/StrangeLoopGames/EcoIssues/files/13954832/DxDiag.txt)

**Comment by D3nnis3n** [comment]
Hello, @Egorimus, in your log there is an access warning:

Uploading Crash Report
UnauthorizedAccessException: Access to the path "C:\Users\sirta\AppData\LocalLow\Strange Loop Games\Eco\inputManager.cfg" is denied.
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x00000] in <00000000000000000000000000000000>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options) [0x00000] in <00000000000000000000000000000000>:0 
  at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize) [0x00000] in <00000000000000000000000000000000>:0 
  at System.IO.StreamWriter..ctor (System.String path) [0x00000] in <00000000000000000000000000000000>:0 
  at System.IO.File.WriteAllText (System.String path, System.String contents) [0x00000] in <00000000000000000000000000000000>:0
  
 Something is blocking access to that file, could be your system setup or any sort of antivirus or security software. Make sure that it is accessible.

**Comment by Egorimus** [comment]
So, when I went to the file location that you listed above, a handful of files in that directory were marked as Hidden for some reason?  (In fact, on closer inspection, many files on my primary drive now have the Hidden attribute when they wouldn't normally possess it including documents and things like that, oddly enough).  Anyway, I toggled off Hidden, and tried creating a world both in 9.7.13 and 10.0.4, and that actually fixed each of my issues!  Thank you for your help on this, D3nnis3n.
